### ISSUE TITLE
RepeatingList might not be accessible

### ISSUE THREAD

**Issue by robere2** [robere2/Portfolio#30]
The `RepeatingList` probably is not accessible, since it is adding and removing elements to and from the DOM. An invisible list should be available for screen readers instead.
